[IT] Update `Mascots` (#9671)
ppy/osu-wiki: wiki/Mascots/it.md

# Mascotte

*Vedi anche: [Mascotte/Galleria](/wiki/Mascots/Gallery)*

Un video di YouTube che mostra le mascotte di osu! può essere visto su: [Mascotte Showcase](https://youtu.be/mJF2cAs_MrI).

## Ufficiali

### ![icona Osu!](/wiki/shared/mode/osu.png) pippi

![pippi](img/pippi.png "pippi")

> pippi, stilizzata con una "p" minuscola, è la mascotte della modalità di gioco osu! dal 2008-07. È anche conosciuta come pippidon in osu!taiko e appare in [Yandere Simulator](https://yanderesimulator.com) come un NPC. La concept art iniziale è stata creata da [Sarumaru](https://osu.ppy.sh/users/9427), la grafica é stata creata da [crystalsuicune](https://osu.ppy.sh/users/9974), e il disegno attuale é stato progetto da [Daru](https://osu.ppy.sh/users/32480).

Reviewer: ```suggestion
pippi, stilizzata con una "p" minuscola, è la mascotte della modalità di gioco osu! dal 2008-07. È anche conosciuta come pippidon in osu!taiko e appare in [Yandere Simulator](https://yanderesimulator.com) come un NPC. La concept art iniziale è stata creata da [Sarumaru](https://osu.ppy.sh/users/9427), la grafica è stata creata da [crystalsuicune](https://osu.ppy.sh/users/9974), e il disegno attuale è stato progetto da [Daru](https://osu.ppy.sh/users/32480).
```

![Yuzu](img/Yuzu.png "Yuzu")

Yuzu é la mascotte di osu!catch che si é unita il 22 Giugno 2014. É nata il 10 Aprile 2000, alta 172cm, e pesa 65kg. Il suo disegno attuale é stato creato da [Thievley](https://osu.ppy.sh/users/4717672). Mentre il suo design originale é stato creato da [ztrot](https://osu.ppy.sh/users/6347); Daru ha creato il disegno relativo al comboburst.

### ![icona osu!mania ](/wiki/shared/mode/mania.png) Mani & Mari

*Per ulteriori informazioni, consulta: [Presentiamo Mani e Mari, le nuove mascotte di osu!mania.](https://osu.ppy.sh/home/news/2020-09-17-introducing-mani-mari-osumania)*

![Mani & Mari](https://assets.ppy.sh/media/mari-mani/wiki-key-condensed.png "Mani & Mari")

Disegnati da [xiemon](https://osu.ppy.sh/users/5203667) dopo essere stati selezionati nella [Most Manic Art Contest](https://osu.ppy.sh/community/contests/80) e annunciati come le mascotte ufficiali di osu!mania, Mani e Mari sono gemelli che non riescono a sopportarsi a vicenda, ma sono più simili di quanto vorrebbero ammettere.

> Mani è un po' un ribelle, sempre alla ricerca di nuovi stili e cose da esplorare, mentre sua sorella Mari (precedentemente conosciuta come Maria) è una classica perfezionista, rigida con una predilezione per i riflettori. Vanno d'accordo come l'olio e l'acqua.

Reviewer: ```suggestion
Mani è un po' un ribelle, sempre alla ricerca di nuovi stili e cose da esplorare, mentre sua sorella Mari (precedentemente conosciuta come Maria) è una classica perfezionista, rigida e le piace stare al centro dell'attenzione. Vanno d'accordo come l'olio e l'acqua.
```

### ![icona osu!taiko ](/wiki/shared/mode/taiko.png) Mocha

*Per il post nelle notizie, consulta: [La nuova mascotte di osu!taiko é qui!](https://osu.ppy.sh/home/news/2017-05-25-the-new-osutaiko-mascot-is-here)*

![Mocha](img/Mocha.png "Mocha")

> Mocha é la mascotte di osu!taiko. Inizialmente é stata disegnata durante il [sesto concorso di fantart](https://osu.ppy.sh/community/contests/2) da [Crowie](https://osu.ppy.sh/users/6894067), che si é classificato 21esimo.

Reviewer: ```suggestion
Mocha è la mascotte di osu!taiko. Inizialmente è stata disegnata durante il [sesto concorso di fantart](https://osu.ppy.sh/community/contests/2) da [Crowie](https://osu.ppy.sh/users/6894067), che si è classificato 21esimo.
```

> ## Cameo

Reviewer: ```suggestion
## Camei
```
Not really sure about it

### Ryūta Ippongi

![Ryuuta](img/Ryuuta.png "Ryuuta")

> >Leader appassionato della squadra delle cheerleader. Ha un'anima generosa, aiuta chi é in difficoltá confortandoli e lottando per loro.

Reviewer: ```suggestion
>Leader della squadra delle cheerleader. Ha un'anima generosa, aiuta chi è in difficoltà confortandoli e lottando per loro.
```

> 一本木龍太 (Ryūta Ippongi) ha raccolto frutta in osu!catch dal 2008 fino al 2014, dove poi é stato sostituito da [Yuzu](#yuzu). É stato creato dalla [iNiS Corporation](https://en.wikipedia.org/wiki/INiS) e faceva parte del vecchio sito web. Compare anche in [Yandere Simulator](https://yanderesimulator.com) come un NPC.

Reviewer: ```suggestion
一本木龍太 (Ryūta Ippongi) ha raccolto frutta in osu!catch dal 2008 fino al 2014, dove poi è stato sostituito da [Yuzu](#yuzu). È stato creato dalla [iNiS Corporation](https://en.wikipedia.org/wiki/INiS) e faceva parte del vecchio sito web. Compare anche in [Yandere Simulator](https://yanderesimulator.com) come un NPC.
```

Ryuuta fa parte della skin di [LuigiHann](https://osu.ppy.sh/users/1079), [Elite Beat osu! HD (1.0 Complete!)](https://osu.ppy.sh/community/forums/topics/190357).

### Agente J

![Agente J](img/Agent_J.png "Agente J")

> Esperto in molto stili di danza che vanno dall'hip-hop al balletto, J puó incantare qualsiasi essere vivente.

> Agente J, conosciuto anche come BA-2 (Beat Agent-2) o J, é stata una delle mascotte di osu! dal 2008, per poi ritirarsi nel 2014, E'stato creato dalla iNiS Corporation e faceva parte del vecchio sito web.

Reviewer: ```suggestion
Agente J, conosciuto anche come BA-2 (Beat Agent-2) o J, è stata una delle mascotte di osu! dal 2008 fino al 2014. È stato creato dalla iNiS Corporation e faceva parte del vecchio sito web.
```

Agente J fa parte della skin di [LuigiHann](https://osu.ppy.sh/users/1079), [Elite Beat osu! HD (1.0 Complete!)](https://osu.ppy.sh/community/forums/topics/190357).

### Don

![Don](img/Don.png "Don")

> > Don é il protagonista della serie [Taiko no Tatsujin](https://en.wikipedia.org/wiki/Taiko_no_Tatsujin). Ë un tamburo taiko con il bordo bianco e quattro gambe, un viso rosso (che sembra di un arancione acceso), e un corpo blu chiaro. Il sogno di Don é condividere la bellezza di Taiko con il resto del mondo. Sono passati 3 anni da quando si é trasferito nelle Wada House, e é diventato abbastanza popolare nella cittadina. Ha un appetito mostruoso, e alcune volte fa spese costose nella Wada House, il che può portare al peggio. Molto spesso finisce le sue frasi con "Ta-don", che significa "Ba-dum" in Giapponese.

Reviewer: ```suggestion
> Don è il protagonista della serie [Taiko no Tatsujin](https://en.wikipedia.org/wiki/Taiko_no_Tatsujin). È un tamburo taiko con il bordo bianco e quattro gambe, un viso rosso (che sembra di un arancione acceso), e un corpo blu chiaro. Il sogno di Don è condividere la bellezza di Taiko con il resto del mondo. Sono passati 3 anni da quando si è trasferito nelle Wada House, ed è diventato abbastanza popolare nella cittadina. Ha un appetito mostruoso, e alcune volte fa spese costose nella Wada House, il che può portare al peggio. Molto spesso finisce le sue frasi con "Ta-don", che significa "Ba-dum" in Giapponese.
```

> 和田どん (Wada Don), conosciuto anche come Don o Don-chan, é stata una della mascotte di osu!taiko da Maggio 2008. É alto 48cm e pesa piú di 100kg. Compare nella skin di osu!taiko. Il suo disegno é stato fatto da Yukiko Yokoo (横尾有希子) e gli é stata data una voce da Narahashi Miki (楢橋 美紀).

Reviewer: ```suggestion
和田どん (Wada Don), conosciuto anche come Don o Don-chan, è stata una della mascotte di osu!taiko da Maggio 2008. È alto 48cm e pesa più di 100kg. Compare nella skin di osu!taiko. Il suo disegno è stato fatto da Yukiko Yokoo (横尾有希子) e gli è stata data una voce da Narahashi Miki (楢橋 美紀).
```

## Community

### Aiko

![Aiko](img/Aiko.png "Aiko")

> Disegnata da [JMC](https://osu.ppy.sh/users/774010), Aiko era una delle partecipanti del design contest delle mascotte di osu!taiko. É una ragazza energica con una passione per osu!taiko, anche se non é la migliore! Indossando un paio di "Tabi" eleganti e una serie di accessori di Pippidon, la mascotte del passato vive in questa ragazza piena d'energia.É piuttosto bassa, con soli 154cm di altezza, ed é nata il 6 aprile 1999.

Reviewer: ```suggestion
Disegnata da [JMC](https://osu.ppy.sh/users/774010), Aiko era una delle partecipanti del design contest delle mascotte di osu!taiko. È una ragazza energica con una passione per osu!taiko, anche se non è la migliore! Indossando un paio di "Tabi" eleganti e una serie di accessori di Pippidon, la mascotte del passato vive in questa ragazza piena d'energia. È piuttosto bassa, con soli 154cm di altezza, ed è nata il 6 aprile 1999.
```

### Alisa

![Alisa](img/Alisa.png "Alisa")

> Disegnata da [\[ Glitch \]](https://osu.ppy.sh/users/3781400), Alisa era una delle partecipanti del design contest delle mascotte di osu!taiko. Gioca ad osu!taiko fin da bambina. Adora suonare canzoni per gli altri, ama la musica, e quando non sta dormendo o mangiando, gioca ad osu!taiko oppure a qualche videogioco retró per divertimento!

Reviewer: ```suggestion
Disegnata da [\[ Glitch \]](https://osu.ppy.sh/users/3781400), Alisa era una delle partecipanti del design contest delle mascotte di osu!taiko. Gioca ad osu!taiko fin da bambina. Adora suonare melodie per gli altri e ama la musica. Quando non sta dormendo o mangiando, gioca ad osu!taiko oppure a qualche videogioco retró per divertimento!
```

### Chirou

![Chirou](img/Chirou.png "Chirou")

> Disegnata da [pyun](https://osu.ppy.sh/users/981534), Chirou era una delle partecipanti del design contest delle mascotte di osu!taiko. É molto severa e esigente, una perfezionista che non ama commettere errori soprattutto mentre gioca a osu!taiko. Ma al di lá del suo aspetto severo, se riesci a raggiungere il suo lato dolce, puó essere gentile e premurosa. Ha 14 anni, nata il 25 Ottobre, gruppo sanguigno AB. É alta 150cm e pesa 45Kg. I suoi hobby includono esercitarsi a suonare la batteria, stare al caldo con la sua coperta e collezionare gemme o rocce. Chirou è apparsa in diverse fanart create dagli utenti.

Reviewer: ```suggestion
Disegnata da [pyun](https://osu.ppy.sh/users/981534), Chirou era una delle partecipanti del design contest delle mascotte di osu!taiko. È molto severa e esigente, una perfezionista che non ama commettere errori soprattutto mentre gioca ad osu!taiko. Ma al di là del suo aspetto severo, se riesci a raggiungere il suo lato dolce, può essere gentile e premurosa. Ha 14 anni, nata il 25 Ottobre, gruppo sanguigno AB. È alta 150cm e pesa 45Kg. I suoi hobby includono esercitarsi a suonare la batteria, stare al caldo con la sua coperta e collezionare gemme o rocce. Chirou è apparsa in diverse fanart create dagli utenti.
```
